tests/core/smoke/main.fmf

```diff
@@ -1,4 +1,5 @@
 summary: Just a basic smoke test
+contact: Some Body <somebody@somewhere.org>
```

Comment:
These is a real `tmt` test, valid metadata, please don't modify it with testing data.

Reply:
Sure, `contact` line is removed.